tfx_addons/sampler/example/local_notebook.ipynb

```diff
@@ -0,0 +1,9670 @@
+{
```

Comment:
<p>what do you think of adding another statistics_gen after sampling, would that help showing the sampling effect?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tfx-addons/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>